src/NuGet.Core/NuGet.Commands/project.json

```diff
@@ -1,86 +1,81 @@
-        "../../../submodules/FileSystem/src/**/AssemblyInfo.cs"
-      ]
-    }
-  },
-  "dependencies": {
-    "NuGet.Client": {
-      "target": "project"
-    },
-    "NuGet.ContentModel": {
-      "target": "project"
-    },
-    "NuGet.ProjectModel": {
-      "target": "project"
-    },
-    "NuGet.Configuration": {
-      "target": "project"
-    },
-    "NuGet.DependencyResolver": {
-      "target": "project"
-    },
-    "NuGet.RuntimeModel": {
-      "target": "project"
-    }
-  },
-  "frameworks": {
-    "net45": {
-      "imports": [
-        "dotnet5.6",
-        "dnxcore50",
-        "portable-net45+win8"
-      ],
-      "frameworkAssemblies": {
-        "System.Xml": "",
-        "System.Xml.Linq": ""
-      },
-      "buildOptions": {
-        "define": [
-          "IS_DESKTOP"
-        ]
-      }
-    },
-    "netstandard1.3": {
-      "imports": [
-        "dotnet5.6",
-        "dnxcore50",
-        "portable-net45+win8"
-      ],
-      "dependencies": {
-        "NETStandard.Library": "1.6.0",
-        "System.Xml.XDocument": "4.0.11"
-      },
-      "buildOptions": {
-        "define": [
-          "IS_CORECLR"
-        ]
-      }
-    }
-  }
-}
+﻿{
+  "version": "3.5.0-*",
+  "copyright": "Copyright .NET Foundation. All rights reserved.",
+  "packOptions": {
+    "licenseUrl": "https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt",
+    "projectUrl": "https://github.com/NuGet/NuGet.Client"
+  },
+  "description": "Complete commands common to command-line and GUI NuGet clients",
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "xmlDoc": true,
+    "nowarn": [
+      "CS1591",
+      "CS1574",
+      "CS1573",
+      "CS1584",
+      "CS1658"
+    ],
+    "compile": {
+      "include": [
+        "../../../Shared/*.cs",
+        "../../../submodules/FileSystem/src/Microsoft.Extensions.FileSystemGlobbing/**/*.cs",
+        "../../../submodules/FileSystem/src/Microsoft.AspNet.FileProviders.Abstractions/**/*.cs",
+        "../../../submodules/FileSystem/src/Microsoft.AspNet.FileProviders.Sources/**/*.cs",
+        "../../../submodules/Common/src/Microsoft.Extensions.Primitives/IChangeToken.cs"
+      ],
+      "exclude": [
+        "../../../submodules/FileSystem/src/**/AssemblyInfo.cs"
+      ]
+    }
+  },
+  "dependencies": {
+    "NuGet.Client": {
+      "target": "project"
+    },
+    "NuGet.ContentModel": {
+      "target": "project"
+    },
+    "NuGet.ProjectModel": {
+      "target": "project"
+    },
+    "NuGet.Configuration": {
+      "target": "project"
+    },
+    "NuGet.DependencyResolver": {
+      "target": "project"
+    },
+    "NuGet.RuntimeModel": {
+      "target": "project"
+    }
+  },
+  "frameworks": {
+    "net45": {
+      "frameworkAssemblies": {
+        "System.Xml": "",
+        "System.Xml.Linq": ""
+      },
+      "buildOptions": {
+        "define": [
+          "IS_DESKTOP"
+        ]
+      }
+    },
+    "netstandard1.3": {
+      "imports": [
```

Comment:
Thanks, I must have missed removing them for that project. There were just a few scenarios that needed imports still.

I should have started with your PR for the imports :smile:  There were just a few differences in that 6.0.4 is still needed for the net45 projects.